config/install/contacts.contact_tab.summary.yml

```diff
@@ -6,10 +6,13 @@ label: Summary
 path: summary
 weight: -99
 relationships: {  }
-block:
-  id: contact_summary_tab
-  label: ''
-  provider: contacts
-  label_display: '0'
-  context_mapping:
-    user: user
+blocks:
+  contact_summary_tab:
+    id: contact_summary_tab
+    label: ''
+    provider: contacts
+    label_display: '0'
+    region: left
+    weight: 0
+    context_mapping:
+      user: user
```

Comment:
This should probably use the three blocks directly, Indiv/Org on left, Notes on right. We can then remove the summary tab block/theme/templates etc and be rid of them!